src/utils/stamped_hashtable.mli

```diff
@@ -0,0 +1,13 @@
+type ('a, 'b) t
```

Comment:
I think we could add a bit more documentation to this module and the reason it exists.

Reply:
Will do :).